test/e2e/upgradetest/recovery_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2017 The etcd-operator Authors
```

Comment:
It's not worth to create one file for each test.
Let's put this onto upgrade_test.go